src/conduits/RwaOutputConduit3.t.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-FileCopyrightText: © 2022 Dai Foundation <www.daifoundation.org>
 // SPDX-License-Identifier: AGPL-3.0-or-later
 //
+// Copyright (C) 2021-2022 Dai Foundation
+//
```

Comment:
```suggestion
```

Reply:
You should probably delete these lines.
This is the old format, we're using the new one in the first line.

PE mentioned in their call that we should include Lev's name whenever applicable, but we can do so using the new format.

Reply:
Done